### PR TITLE
Fwdport forward/rel/6.4 to master

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -74,7 +74,8 @@
     "perf": "mocha --compilers js:babel-core/register tests/performance/s3standard.js",
     "start": "node init.js && node index.js",
     "start_utapi": "node utapiServer.js",
-    "test": "S3BACKEND=mem mocha --compilers js:babel-core/register --recursive tests/unit && S3_CONFIG_FILE='./tests/unit/testConfigs/legacyTestConfig.json' S3BACKEND=mem mocha --compilers js:babel-core/register --recursive tests/unit",
```
Comment (bennettbuchanan) [inline package.json]: We should instead be keeping this test script in master, correct?

Comment (dora-korpar) [inline package.json]: Yup! That makes sure that both the new and legacy configs are tested :)

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
@ironman-machine
SCALITY_UTAPI_BRANCH=fwdport/6.4-master
try

**Comment by bennettbuchanan** [comment]
@ironman-machine try

**Comment by bennettbuchanan** [comment]
e2e passed

**Pull request merged**